<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QMainWindow" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shuffleButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Shuffle</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bubbleSort">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bubble sort</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mergeSort">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Merge sort</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="quickSort">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>470</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Quicksort</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radixSort">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>470</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Radix sort</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="barCount">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="fitCheckBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fit bars to area</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="delayBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Bars</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay (ms)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="coctailShakerSort">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Coctail shaker sort</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="heapSort">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Heap sort</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
